### ISSUE TITLE
Not working for me

### ISSUE THREAD

**Issue by ibakayoko** [tfutils/tfenv#151]
Mac-C0007380:~ ib$ brew install tfenv
Updating Homebrew...
==> Downloading https://github.com/tfutils/tfenv/archive/v1.0.2.tar.gz
Already downloaded: /Users/ixb/Library/Caches/Homebrew/downloads/57d000da62948b9bc641a5c4d5aeb1b82c7951025a1c6205baf0ffcc1c2cc5d3--tfenv-1.0.2.tar.gz
🍺  /usr/local/Cellar/tfenv/1.0.2: 20 files, 29.2KB, built in 7 seconds
Mac-C0007380:~ ib$ tfenv
tfenv 1.0.2
Usage: tfenv <command> [<options>]

Mac-C0007380:~ ib$ tfenv list-remote
Mac-C0007380:~ ib$ tfenv install 0.12
tfenv: tfenv-install: [ERROR] No versions matching '0.12' found in remote
Mac-C0007380:~ ib$ tfenv install latest
tfenv: tfenv-install: [ERROR] No versions matching 'latest' found in remote

**Comment by davidji99** [comment]
You might need to do `tfenv install 0.12.0` (add the patch version).

**Comment by Zordrak** [comment]
Or alternatively `tfenv install latest:^0.12`

**Comment by ibakayoko** [comment]
Still not working

Mac-C0007380:~ ib$ terraform
tfenv: tfenv-version-name: [ERROR] No versions of terraform installed. Please install one with: tfenv install
/usr/local/Cellar/tfenv/1.0.2/libexec/tfenv-exec: line 22: /usr/local/Cellar/tfenv/1.0.2/versions//terraform: No such file or directory

Mac-C0007380:~ ib$ tfenv install 0.12
tfenv: tfenv-install: [ERROR] No versions matching '0.12' found in remote

Mac-C0007380:~ ib$ tfenv install 0.12.0
tfenv: tfenv-install: [ERROR] No versions matching '0.12.0' found in remote

Mac-C0007380:~ ib$ tfenv install latest:^0.12
tfenv: tfenv-install: [ERROR] No versions matching 'latest:^0.12' found in remote

**Comment by Zordrak** [comment]
Difficult to say what's going on without a little more debug output. Perhaps if you run with bash -x it might become a little clearer.

Alternatively, can I recommend v2.0.0-alpha3 which not only works a little differently, but has several debug levels available, e.g. TFENV_DEBUG=1 tfenv install latest

**Comment by Zordrak** [comment]
Please can you test the latest master? Not only has v2.0.0 now been released, there are several MacOS compatibility fixes in the master branch due for release soon.

If you encounter any problems please provide a log with  TFENV_DEBUG=1

**Comment by alechp** [comment]
For anyone else having this issue:
```
cat: /usr/local/Cellar/tfenv/2.0.0/version: No such file or directory
Version could not be resolved (set by /usr/local/Cellar/tfenv/2.0.0/version or tfenv use <version>)
```

You'll need to specify `tfenv use <version>`

Example of failure:
![Screen Shot 2020-08-12 at 11 30 24 AM](https://user-images.githubusercontent.com/3058939/90053357-396b7880-dc8f-11ea-9177-6b417b952aa9.png)

All commands used (in order):
```sh
brew install tfenv
tfenv install 0.x.x
tfenv use 0.x.x
terraform init
```

**Comment by fundener** [comment]
Good solution @alechp, thanks, it helped.
In my case without checking what is the latest available version number:

```
brew install tfenv
tfenv install latest
tfenv use latest
terraform init
```
